src/Player/LavaTrack.cs

```diff
@@ -128,6 +128,19 @@ public class LavaTrack {
             Position = TimeSpan.FromMilliseconds(position);
         }
 
+        internal void Update(LavaTrack lavaTrack) {
```

Comment:
Any reason why update method is needed? It's being used only once.

Reply:
It's not really needed but added it just to make it satisfied since it's being decoded when track start